src/ucp/stream/stream_send.c

```diff
@@ -86,17 +91,51 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_stream_send_nb,
                  ucp_ep_h ep, const void *buffer, size_t count,
                  uintptr_t datatype, ucp_send_callback_t cb, unsigned flags)
 {
+    ucp_request_param_t param = {
```

Comment:
the same as above, better to separate struct var definition and initialization

Reply:
it's a common way to implement legacy API via NBX